.env

```diff
@@ -1 +1,3 @@
 SLACK_CONNECT_HREF=https://slack.com/oauth/authorize?scope=incoming-webhook&client_id=218633036018.221954264375
+SLACK_API_URL=http://localhost:8001
```

Comment:
as mentioned yesterday, development specific URLs go into .env.development, production ones in .env.production

Reply:
I had to step out to a meeting during that session so I didn't catch that. I'll take care of it.